chia-blockchain/defaults/main.yml

```diff
@@ -263,13 +262,13 @@ nameserver: chia.net.
 ttl: 300
 soa_rname: "hostmaster.chia.net."
 
-# Vector/Loki Related
-add_vector_config: false
+# promtail/Loki Related
+add_promtail_config: false
```

Comment:
Do we even need this as part of the chia role now? There's no config being added by this role, so I think the way to actually do this is just call out promtail specifically in the playbook, and remove essentially all references from here?

Reply:
removed